apps/backoffice-v2/src/lib/blocks/hooks/useDirectorsBlocks/useDirectorsBlocks.tsx

```diff
@@ -335,11 +407,6 @@ export const useDirectorsBlocks = ({
                     type: 'heading',
                     value: `Director - ${director.firstName} ${director.lastName}`,
                   })
-                  .addCell({
-                    id: 'actions',
-                    type: 'container',
-                    value: getReuploadActionOrBadge(workflow?.tags ?? []),
-                  })
```

Comment:
I don't understand why this block was removed?

Reply:
First iteration used global actions for all documents. Now each document has its own actions.